File: boilerplate/backend/node/config/passwordless.ts
import Passwordless from "supertokens-node/recipe/passwordless";
import Session from "supertokens-node/recipe/session";

export const recipeList = [
    Passwordless.init({
        contactMethod: "EMAIL_OR_PHONE",
        flowType: "USER_INPUT_CODE_AND_MAGIC_LINK",
    }),
    Session.init(),
]

File: boilerplate/backend/python-flask/config/passwordless.py
from supertokens_python.recipe import passwordless, session
from supertokens_python.recipe.passwordless import ContactEmailOrPhoneConfig
from supertokens_python import (
    InputAppInfo,
    SupertokensConfig,
)

supertokens_config=SupertokensConfig(connection_uri="https://try.supertokens.io")

app_info=InputAppInfo(
    app_name="Supertokens",
    api_domain="http://localhost:3001",
    website_domain="http://localhost:3000",
)

framework="flask"

recipe_list=[
    session.init(),
    passwordless.init(
        flow_type="USER_INPUT_CODE_AND_MAGIC_LINK",
        contact_config=ContactEmailOrPhoneConfig()
    )
]